fastapi_sso/sso/linkedin.py

```diff
@@ -0,0 +1,43 @@
+"""LinkedIn SSO Oauth Helper class"""
+
+from typing import TYPE_CHECKING, Optional
+
+from fastapi_sso.sso.base import DiscoveryDocument, OpenID, SSOBase
+
+if TYPE_CHECKING:
+    import httpx
+
+
+class LinkedInSSO(SSOBase):
+    """Class providing login via LinkedIn SSO"""
+
+    provider = "linkedin"
+    scope = ["openid", "profile", "email"]
+    additional_headers = {"accept": "application/json"}
+
+    async def get_discovery_document(self) -> DiscoveryDocument:
+        return {
+            "authorization_endpoint": "https://www.linkedin.com/oauth/v2/authorization",
+            "token_endpoint": "https://www.linkedin.com/oauth/v2/accessToken",
+            "userinfo_endpoint": "https://api.linkedin.com/v2/userinfo",
+        }
+
+    async def openid_from_response(self, response: dict, session: Optional["httpx.AsyncClient"] = None) -> OpenID:
+        """Response format
+        sub	Subject Identifier	Text
+        name	Full name	Text
+        given_name	Member's first name	Text
+        family_name	Member's last name	Text
+        picture	Member's profile picture URL	Text
+        locale	Member's locale	dict {'country': str, 'language': str}
+        email	Member's primary e-mail address.	Text
+        email_verified	Indicator that Member's primary email has been verified	Boolean
```

Comment:
I don't think this is neccessary, maybe just link the docs if this is copied from there? If this is something you had to find out yourself, I'd probably prefer this in a comment rather than a docstring.

Reply:
I've removed this doc string

Reply:
> I don't think this is neccessary, maybe just link the docs if this is copied from there? If this is something you had to find out yourself, I'd probably prefer this in a comment rather than a docstring.

Hi @tomasvotava,

I've removed that docstring.